up42/catalog.py

```diff
@@ -129,15 +119,22 @@ def construct_parameters(
             squash_multiple_features="footprint",
         )
 
-        # TODO: cc also contains nan with sentinel 1 etc. ignore?
+        if ascending:
+            sort_order = "asc"
```

Comment:
Minor Suggestion: `sort_order = "asc" if ascending else "desc"`

Reply:
Not the biggest fan of oneliners but here it seems fitting, thanks